packages/api/src/server.ts

```diff
@@ -41,34 +43,34 @@ export class WepublishServer {
       if (mtp.key in dbAdapter) {
         const dbAdapterKeyTyped = mtp.key as keyof typeof dbAdapter
         mtp.methods.forEach(method => {
+          const adapter = dbAdapter[dbAdapterKeyTyped] as Record<string, any>
```

Comment:
Extracted `dbAdapter[dbAdapterKeyTyped]` to type it once and not have to repeat it.
As all the DB Adapters do not share the same keys, the object keys are being typed as `string`,  as all of the methods are different but not unknown, they have to be `any` it seems. They could also be typed to `Function | any`, but since `any` is broader and includes `Function`, it would simply resolve into `any` regardless.